tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -354,6 +354,15 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("routetemplate_expansion_enabled");
                     writer.WriteValue(instanceSettings.ExpandRouteTemplatesEnabled);
 
+                    writer.WritePropertyName("enable_querystring_reporting");
```

Comment:
And here too, to match our other naming conventions this could be named to `querystring_reporting_enabled`